apps/sports-helsinki/src/domain/event/eventKeywords/EventKeywords.tsx

```diff
@@ -45,22 +39,8 @@ const EventKeywords: React.FC<Props> = ({
     locale
   );
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const handleClick =
-    (type: 'dateType' | 'isFree' | 'text', value = '') =>
-    () => {
-      const search = getSearchQuery({
-        ...EVENT_DEFAULT_SEARCH_FILTERS,
-        dateTypes: type === 'dateType' ? [value] : [],
-        isFree: type === 'isFree',
-        q: type === 'text' ? [value] : [],
-      });
-
-      router.push(
-        `${routerHelper.getI18nPath(ROUTES.SEARCH, locale)}${search}`
-      );
-      scrollToTop();
-    };
+  const handleClick = (type: 'dateType' | 'isFree' | 'text', value = '') =>
```

Comment:
This "easily" goes out of sync. I think that the best thing would be that the `getKeywordOnClickHandler` would have a type that it uses and the same property-type could be used here too. But, currently the Typescript allows this also: 

```suggestion
const handleClick = (type: Parameters<typeof getKeywordOnClickHandler>[2], value = '') =>
```

Reply:
As https://github.com/City-of-Helsinki/events-helsinki-monorepo/pull/294#discussion_r1163625577